### PR TITLE
Update helpers.py

### PR DESCRIPTION
**Pull request by daroclark**
added the clear_target_folder function

### CODE REVIEW

File: scripts/helpers/helpers.py [hunk 1]
```diff
@@ -318,3 +318,12 @@ def move_file(bucket, source_path, target_path, filename):
     except Exception as error:
             ## do nothing
             print('Error Occured: rename_file', error)
+
```
Comment (annajgibson) [inline scripts/helpers/helpers.py:321]:
```suggestion
            
            
```

File: scripts/helpers/helpers.py [hunk 2]
```diff
@@ -318,3 +318,12 @@ def move_file(bucket, source_path, target_path, filename):
     except Exception as error:
             ## do nothing
             print('Error Occured: rename_file', error)
+            
+def clear_target_folder(s3_bucket_target):
+    s3 = boto3.resource('s3')
+    folderString = s3_bucket_target.replace('s3://', '')
```
Comment (annajgibson) [inline scripts/helpers/helpers.py:324]:
```suggestion
    folder_string = s3_bucket_target.replace('s3://', '')
```

File: scripts/helpers/helpers.py [hunk 3]
```diff
@@ -318,3 +318,12 @@ def move_file(bucket, source_path, target_path, filename):
...
+    bucketName = folderString.split('/')[0]
```
Comment (annajgibson) [inline scripts/helpers/helpers.py:325]:
```suggestion
    bucket_name = folderString.split('/')[0]
```

File: scripts/helpers/helpers.py [hunk 4]
```diff
@@ -318,3 +318,12 @@ def move_file(bucket, source_path, target_path, filename):
...
+    prefix = folderString.replace(bucketName+'/', '')+'/'
```
Comment (annajgibson) [inline scripts/helpers/helpers.py:326]:
```suggestion
    prefix = folder_string.replace(bucket_name+'/', '')+'/'
```

File: scripts/helpers/helpers.py [hunk 5]
```diff
@@ -318,3 +318,12 @@ def move_file(bucket, source_path, target_path, filename):
...
+    bucket = s3.Bucket(bucketName)
```
Comment (annajgibson) [inline scripts/helpers/helpers.py:327]:
```suggestion
    bucket = s3.Bucket(bucket_name)
```

**Review by annajgibson — COMMENTED** [review]
have renamed variables to lower case and added extra line between previous function, otherwise looks good

**Review by LBHSBALLEY — APPROVED** [review]

**Pull request merged**